docs/customactions.rst

```diff
@@ -104,7 +104,7 @@ which might look like this:
          return "action_check_restaurants"
 
       def run(self, dispatcher, tracker, domain):
-         # type: (Dispatcher, DialogueStateTracker, Domain) -> List[Event]
+         # type: (Dispatcher, Tracker, Domain) -> List[Event]
```

Comment:
Could you please fix other types as well: `# type: (CollectingDispatcher, Tracker, Dict[Text, Any]) -> List[dict]`